src/popup/views/not-authed.tsx

```diff
@@ -1,5 +1,16 @@
 import * as React from 'react'
 
 export function NotAuthed({ signInWithTwitter }: Pick<Dispatchers<UserAction>, 'signInWithTwitter'>): JSX.Element {
-  return <button onClick={signInWithTwitter}>Sign in with twitter</button>
+  return (
+    <div className="not-authed">
+      <img src="../../../img/roar_128.png" width="128px" height="128px" />
```

Comment:
This and the twitter logo should be inlined svgs so that the scaling is always exactly right. You can export the [svg's from Figma](https://www.figma.com/community/plugin/814345141907543603).